docs/library/spi_engine/spi_engine_interconnect.rst

```diff
@@ -54,15 +54,13 @@ Signal and Interface Pins
    * - resetn
      - Synchronous active-low reset.
        Resets the internal state of the module.
-   * - s0_ctrl
+   * - s*_ctrl
```

Comment:
This creates the same warning and situation as the one on the lanes of the jesd transport layer

Reply:
No longer the case on doctools 0.3.15